sharepoint/sharepoint-ps/sharepoint-online/Get-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,134 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spodataaccessgovernanceinsight
+applicable: SharePoint Online
+title: Get-SPODataAccessGovernanceInsight
+schema: 2.0.0
+author: pvrk
+ms.author: pullabhk
+manager: 
+ms.reviewer:
+---
+
+# Get-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+Lists various 'Data Access Governance' (DAG) reports in SharePoint admin center.
+
+## SYNTAX
+
+### GetAllReportsParameterSet
+```
+Get-SPODataAccessGovernanceInsight -ReportEntity <ReportEntityEnum> [-WorkLoad <WorkloadEnum>] [-ReportType <ReportTypeEnum>] [<CommonParameters>]
+```
+
+### GetReportParameterSet
+```
+Get-SPODataAccessGovernanceInsight -ReportID <Guid> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Fetch details of various DAG reports available in SharePoint admin center.
```

Comment:
>Fetch [](http://example.com/codeflow?start=0&length=5)

Fetch -> Fetches

"This cmdlet fetches details of various DAG reports available in SharePoint admin center."